linux_docker_resources/rti_web_binaries_install_script.py

```diff
@@ -26,8 +26,8 @@ def install_connext(installer_path, install_directory):
         while(keep_agreeing):
             result_index = child.expect([
                 'Installation Directory.*?:',
-                'Press \[Enter\] to continue:',
-                'Do you accept this license\? \[y/n\]: '])
+                r'Press \[Enter\] to continue:',
+                r'Do you accept this license\? \[y/n\]: '])
```

Comment:
It's unclear from any commit message why this change was made alongside the break system packages environment variable usage.